scene/gui/base_button.cpp

```diff
@@ -144,7 +144,7 @@ void BaseButton::_toggled(bool p_pressed) {
 }
 
 void BaseButton::on_action_event(Ref<InputEvent> p_event) {
-	if (p_event->is_pressed()) {
+	if (p_event->is_pressed() && status.hovering) {
```

Comment:
This makes it impossible to press buttons with keyboard.

Reply:
Oo! I'll try to fix that.